buildSrc/src/main/java/io/opentelemetry/instrumentation/gradle/AutoInstrumentationPlugin.java

```diff
@@ -29,106 +21,20 @@
 // TODO(anuraaga): Migrate more build logic into this plugin to avoid having two places for it.
```

Comment:
does this PR close this TODO?

Reply:
Don't think so - I previously used a Java plugin for this since it was complicated, and wanted to share the `Constants.java` list of classes here with copy-paste. Now I'll probably move this into the `.gradle` file in a followup since it's barely any code.